### PR TITLE
Shared linting for acceptance tests

### PR DESCRIPTION
**Pull request by kylehar9**
# Description

Add shared configuration for our WDIO acceptance tests. 
Add fixtures to the test overrides list where certain rules are disabled.

## Changes

- `qa.js` configuration added.
- Release version `6.0.0-alpha.4`.

### CODE REVIEW

File: wdio.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  overrides: [
+    {
+      files: ['test/client/**/*.js'],
```
Comment (LToronto) [inline wdio.js:4]: this doesn't work for babylon

Comment (skye2k2) [reply wdio.js:4]: Would just `test/**/*.js` work?

**Review by LToronto — APPROVED** [review]

**Review by FSVetaz — APPROVED** [review]

File: demo/test/snapshots/linting-config.test.js.md [hunk 2]
```diff
@@ -88,9 +88,6 @@ Generated by [AVA](https://ava.li).
         "@typescript-eslint/keyword-spacing": [␊
           "off"␊
         ],␊
-        "@typescript-eslint/lines-around-comment": [␊
```
Comment (skye2k2) [inline demo/test/snapshots/linting-config.test.js.md]: @FSVetaz: I understand running this command, but I am extremely confused as to why this change does not show anywhere in the resulting files changed.

Comment (FSVetaz) [reply demo/test/snapshots/linting-config.test.js.md]: With or without it, nothing changes in the resulting eslint linting/formatting
